src/meta/processors/job/JobManager.h

```diff
@@ -295,11 +295,11 @@ class JobManager : public boost::noncopyable, public nebula::cpp::NonMovable {
   nebula::kvstore::KVStore* kvStore_{nullptr};
   AdminClient* adminClient_{nullptr};
 
-  std::map<GraphSpaceID, std::mutex> muReportFinish_;
+  std::map<GraphSpaceID, std::unique_ptr<std::mutex>> muReportFinish_;
   // Start & stop & finish a job need mutual exclusion
   // The reason of using recursive_mutex is that, it's possible for a meta job try to get this lock
   // in finish-callback in the same thread with runJobInternal
-  std::map<GraphSpaceID, std::recursive_mutex> muJobFinished_;
+  std::map<GraphSpaceID, std::unique_ptr<std::recursive_mutex>> muJobFinished_;
```

Comment:
No concurrent operations on muJobFinished_?

Reply:
there are concurrent operations

Reply:
Referring to the stack information above, the root cause is the problem caused by multi-threaded use of map.